ui/app/components/ui/button-group/index.scss

```diff
@@ -4,7 +4,8 @@
   align-items: center;
 
   &__button {
-    font-size: 1rem;
+    @include H5;
```

Comment:
This unfortunately is breaking the "Customize gas" page; the text is now cut off due to the increase in line height.

![gas-estimate](https://user-images.githubusercontent.com/2459287/89460559-da62ac80-d740-11ea-88bf-2fd1333a483b.png)